Python/crossinterp_data_lookup.h

```diff
@@ -701,6 +701,13 @@ _PyFunction_FromXIData(_PyXIData_t *xidata)
         Py_DECREF(code);
         return NULL;
     }
+    PyThreadState *tstate = _PyThreadState_GET();
+    if (PyDict_SetItemString(globals, "__builtins__",
```

Comment:
In case you want to save a few microseconds:
```suggestion
    if (PyDict_SetItemString(globals, &_Py_ID(__builtins__),
```

Reply:
Yeah, not a big difference, but probably still worth doing.

Reply:
done